code/modules/projectiles/projectile/bullets.dm

```diff
@@ -186,6 +187,28 @@
 		T.part = organ
 		LAZYADD(organ.implants, T)
 
+/obj/item/projectile/bullet/acid
+	name = "acid capsule"
+	damage = 20
+	damage_type = DAMAGE_BURN
+	agony = 30
```

Comment:
these are insane stats for a projectile, nerf them

Reply:
> these are insane stats for a projectile, nerf them

How so? Beanbags are:

`/obj/item/projectile/bullet/shotgun/beanbag		//because beanbags are not bullets
	name = "beanbag"
	check_armor = "melee"
	damage = 10
	agony = 60
	embed = 0
	sharp = 0`

Is this really that out of whack compared to other projectiles? I definitely agree embed and sharp should be set off, though.

Reply:
> Also, does this burn damage delete blood? I'm not able to tell just from looking at it.

Yes, and burns have a higher pain mod + bypass siemens